APMrover2/Log.cpp

```diff
@@ -93,15 +93,13 @@ int8_t Rover::erase_logs(uint8_t argc, const Menu::arg *argv)
 
 int8_t Rover::select_logs(uint8_t argc, const Menu::arg *argv)
 {
-    uint16_t bits;
+    uint16_t bits = 0;
```

Comment:
hum I don't think as it is in a function. the problem is that bits is create and just after set to 0, so it is better to allocate it directly to 0 instead of doing creation + allocation, no?